src/scythe-bidder/create-room.tsx

```diff
@@ -2,37 +2,117 @@
 
 import React from "react";
 import { jsx } from "@emotion/core";
-import { Button, Card, Form, notification, Select, Switch } from "antd";
+import { Button, Card, Form, notification, Select, Modal } from "antd";
 import client from "./client";
-import { FACTIONS_BASE, FACTIONS_IFA, MATS_BASE, MATS_IFA } from "./constants";
+import {
+  FACTIONS_BASE,
+  FACTIONS_IFA,
+  FACTIONS_HI,
+  FACTIONS_LO,
+  MATS_BASE,
+  MATS_IFA,
+  MATS_HI,
+  MATS_LO,
+} from "./constants";
 import { MAX_PLAYERS_BASE, MAX_PLAYERS_IFA, MIN_PLAYERS } from "./constants";
 import { SCYTHE_BIDDER } from "./constants";
+import { QuestionCircleFilled } from "@ant-design/icons";
+
+enum GameSetting {
+  Base,
+  IFA,
+  Hi,
+  Lo,
+}
 
 export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
   const [numPlayers, setNumPlayers] = React.useState(2);
-  const [isIfaActive, setIsIfaActive] = React.useState<boolean>(true);
-  const maxPlayers = isIfaActive ? MAX_PLAYERS_IFA : MAX_PLAYERS_BASE;
+  const [activeCombinations, setActiveCombinations] = React.useState(
+    GameSetting.IFA
+  );
+  const maxPlayers =
+    activeCombinations === GameSetting.IFA ? MAX_PLAYERS_IFA : MAX_PLAYERS_BASE;
+
+  const { Option } = Select;
+
+  const gameSettingLabel = (
+    <div>
+      Game setting{" "}
+      <QuestionCircleFilled
+        css={{ verticalAlign: "50%" }}
```

Comment:
Does this work? I was thinking about making the parent div a flexbox.

Reply:
yes, here is a screen of it
![image](https://user-images.githubusercontent.com/62440222/99014356-2da28300-2518-11eb-84f0-8525f4ab2331.png)